### PR TITLE
Added a line to switch astropy ephemeris via astropy.coordinates.solar_system_ephemeris

### LINKED ISSUE #3114: Add documentation explaining why/how to switch the Astropy ephemeris
Astropy's built-in ephemeris is not as accurate as a JPL ephemeris (e.g., DE430), although the discrepancies are appreciable only when querying non-Earth planets.  Our documentation should explain how to switch the Astropy ephemeris (via `astropy.coordinates.solar_system_ephemeris`).

### PR DESCRIPTION
**Pull request by Jett-Code**
## PR Description

 Fixes #3114

### CODE REVIEW

**Comment by Jett-Code** [comment]
Is this what you meant by linking to astropy page?

File: sunpy/coordinates/ephemeris.py [hunk 1]
```diff
@@ -61,7 +61,9 @@ def get_body_heliographic_stonyhurst(body, time='now', observer=None, *, include
     -----
     There is no correction for aberration due to observer motion.  For a body close to the Sun in
     angular direction relative to the observer, the correction can be negligible because the
-    apparent location of the body will shift in tandem with the Sun.
+    apparent location of the body will shift in tandem with the Sun. As Astropy's built-in ephemeris
+    is not as accurate as JPL ephemeris so you can use `~astropy.coordinates.solar_system_ephemeris`
```
Comment (nabobalis) [inline sunpy/coordinates/ephemeris.py:65]: I assume we would want to link to https://docs.astropy.org/en/stable/coordinates/solarsystem.html as well.

It has the ref, `astropy-coordinates-solarsystem` we should be able to link that directly.

**Comment by nabobalis** [comment]
It would be nice to add an example of changing the ephemerides using `solar_system_ephemeris` but I will leave that up for Albert to chime in on.

File: sunpy/coordinates/ephemeris.py (hunk 1 above)
Comment (Jett-Code) [reply sunpy/coordinates/ephemeris.py:65]: `~astropy.coordinates.solar_system_ephemeris`
I guess clicking on this link will open https://docs.astropy.org/en/stable/coordinates/solarsystem.html .
Doesn't it?

Comment (nabobalis) [reply sunpy/coordinates/ephemeris.py:65]: Did you build the documentation and check? 

I don't think that will link that page but the API page for `solar_system_ephemeris`

Comment (Jett-Code) [reply sunpy/coordinates/ephemeris.py:65]: I got it

File: sunpy/coordinates/ephemeris.py [hunk 2]
```diff
@@ -61,7 +61,9 @@ def get_body_heliographic_stonyhurst(body, time='now', observer=None, *, include
...
+    in order to change the ephemeris. You can refer `~astropy.coordinates.solarsystem` for docs.
```
Comment (nabobalis) [inline sunpy/coordinates/ephemeris.py:66]: This isn't working:
```
/home/runner/work/sunpy/sunpy/.tox/build_docs/lib/python3.9/site-packages/sunpy/coordinates/ephemeris.py:docstring of sunpy.coordinates.ephemeris.get_body_heliographic_stonyhurst:12: WARNING: py:obj reference target not found: astropy.coordinates.solarsystem
/home/runner/work/sunpy/sunpy/.tox/build_docs/lib/python3.9/site-packages/sunpy/coordinates/ephemeris.py:docstring of sunpy.coordinates.ephemeris.get_body_heliographic_stonyhurst:12: WARNING: py:obj reference target not found: astropy.coordinates.solarsystem
```

Comment (Jett-Code) [reply sunpy/coordinates/ephemeris.py:66]: Can you tell me how to check whether the docs are built or not?

Comment (nabobalis) [reply sunpy/coordinates/ephemeris.py:66]: If you can read over: https://docs.sunpy.org/en/latest/dev_guide/contents/documentation.html#sphinx  that will walk you through the steps.

Comment (Jett-Code) [reply sunpy/coordinates/ephemeris.py:66]: Thanks

File: sunpy/coordinates/ephemeris.py [hunk 3]
```diff
@@ -63,7 +63,8 @@ def get_body_heliographic_stonyhurst(body, time='now', observer=None, *, include
     angular direction relative to the observer, the correction can be negligible because the
     apparent location of the body will shift in tandem with the Sun. As Astropy's built-in ephemeris
     is not as accurate as JPL ephemeris so you can use `~astropy.coordinates.solar_system_ephemeris`
-    in order to change the ephemeris. You can refer `~astropy.coordinates.solarsystem` for docs.
+    in order to change the ephemeris. You can refer
+    `astropy.coordinates.solarsystem <https://docs.astropy.org/en/stable/coordinates/solarsystem.html>`__ for docs.
```
Comment (nabobalis) [inline sunpy/coordinates/ephemeris.py:67]: This page should be linkable using sphinx directly and not a url.

It is also not an API page, `astropy.coordinates.solarsystem` is not a astropy module or method.

File: sunpy/coordinates/ephemeris.py [hunk 4]
```diff
@@ -61,7 +61,9 @@ def get_body_heliographic_stonyhurst(body, time='now', observer=None, *, include
     -----
     There is no correction for aberration due to observer motion.  For a body close to the Sun in
     angular direction relative to the observer, the correction can be negligible because the
-    apparent location of the body will shift in tandem with the Sun.
+    apparent location of the body will shift in tandem with the Sun. As Astropy's built-in ephemeris
+    is not as accurate as JPL ephemeris so you can use `~astropy.coordinates.solar_system_ephemeris`
+    in order to change the ephemeris. You can refer :ref:`astropy-coordinates-solarsystem` for docs.
```
Comment (nabobalis) [inline sunpy/coordinates/ephemeris.py:66]:
```suggestion
    apparent location of the body will shift in tandem with the Sun.

    It is important to be aware that Astropy's built-in ephemeris is not as
    accurate as JPL ephemeris, so one can use `~astropy.coordinates.solar_system_ephemeris`
    in order to change the ephemeris. You can refer :ref:`astropy-coordinates-solarsystem` for more information.
```

File: sunpy/coordinates/ephemeris.py [hunk 5]
```diff
@@ -63,6 +63,10 @@ def get_body_heliographic_stonyhurst(body, time='now', observer=None, *, include
     angular direction relative to the observer, the correction can be negligible because the
     apparent location of the body will shift in tandem with the Sun.
 
+    It is important to be aware that Astropy's built-in ephemeris is not as
+    accurate as JPL ephemeris, so one can use `~astropy.coordinates.solar_system_ephemeris`
+    in order to change the ephemeris. You can refer to :ref:`astropy-coordinates-solarsystem` for more information.
```
Comment (ayshih) [inline sunpy/coordinates/ephemeris.py:68]:
```suggestion
    For planets other than Earth, Astropy's built-in ephemeris is not as accurate as JPL
    ephemerides, so one can use :func:`astropy.coordinates.solar_system_ephemeris` to switch
    to a JPL ephemeris. See :ref:`astropy-coordinates-solarsystem` for more information, and see
    :ref:`sphx_glr_generated_gallery_units_and_coordinates_venus_transit.py` for an example.
```

**Review by ayshih — CHANGES_REQUESTED** [review]

**Comment by nabobalis** [comment]
I don't understand why this was closed but I reopened it as I think once Albert reviews it again, it is good to go.

**Review by ayshih — APPROVED** [review]
Looks good, squash when merging

**Comment by nabobalis** [comment]
Thanks @Jett-Code for the PR and thank you @ayshih for your review.

**Comment by Jett-Code** [comment]
> Thanks @Jett-Code for the PR and thank you @ayshih for your review.

Thank you @nabobalis and @ayshih for guiding me throughout the issue.

**Pull request merged**